src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/enums/SessionConflict.kt

```diff
@@ -3,5 +3,5 @@ package uk.gov.justice.digital.hmpps.visitscheduler.dto.enums
 @Suppress("unused")
 enum class SessionConflict {
   NON_ASSOCIATION,
-  DOUBLE_BOOKED,
+  DOUBLE_BOOKING_OR_RESERVATION,
```

Comment:
If we're renaming this (and I do think the existing name is misleading), then does `EXISTING_BOOKING_OR_RESERVATION` better describe it? For me 'double' implies that we've **already** got two there - but isn't it actually that there is **one** there and it's warning you of the risk of a second?